src/fn/move-contacts.js

```diff
@@ -21,6 +22,29 @@ module.exports = {
   }
 };
 
+const moveReportsBatch = async (db, contactIds, skip, replacementLineage, contactId, options) => {
+  const queryOptions = {
+    keys: contactIds.map(id => [`contact:${id}`]),
+    include_docs: true,
+    limit: BATCH_SIZE,
+    skip: skip,
+  };
+  const result = await db.query('medic-client/reports_by_freetext', queryOptions);
+  const reportDocs = result.rows.map(row => row.doc);
+  info(`Processing ${skip} to (${skip + BATCH_SIZE}) docs of ${result.total_rows} total`);
+  const updatedReports = replaceLineageInReports(reportDocs, replacementLineage, contactId);
+  updatedReports.forEach(updatedDoc => {
+    lineageManipulation.minifyLineagesInDoc(updatedDoc);
```

Comment:
This code is duplicated. Please separate this in another function so we don't have to call `minifyLineagesInDoc` or `writeDocumentToDisk` in multiple places.